R/to_xml.R

```diff
@@ -7,6 +7,9 @@
 #' @param sourcepos passed to [commonmark::markdown_xml()]. If `TRUE`, the
 #'   source position of the file will be included as a "sourcepos" attribute.
 #'   Defaults to `FALSE`.
+#' @param anchor_links if `TRUE`, anchor links (in the style of
```

Comment:
maybe mention what happened to them otherwise (they don't disappear, they're merged into the link(s)).